pkg/loop/internal/test/README.md

```diff
@@ -0,0 +1,95 @@
+This package contains test implementations for all the components that compose OCR2 and OCR3 LOOPPs.
+
+The directory layout is intended to mirror the natural interfaces composition of libocr:
+- ocr2: contains static test implementations for reuseable reporting plugin, factory, and plugin provider
+- ocr3: implements the ocr3 generic-based reporting plugin and reused common components from ocr2
+- resources: represents core-node provided resources used to in calls to core node api(s) eg pipeline_runner, keystore, etc
```

Comment:
Did you consider other names? `core` seems straightforward to me :shrug: 
```suggestion
- core: represents core-node provided resources used to in calls to core node api(s) eg pipeline_runner, keystore, etc
```
Or would it make sense to split them up further? e.g. if we promote the keystore to a plugin later, outside of core-node provided things, will the organization still make sense?

Reply:
i originally had all the `resources` in there own packages. that felt too granular at this stage because it led to tiny packages. i consider `core` but decided to be a little more agnostic precisely because of questions about things like the keystore.